operators/constellation-node-operator/internal/azure/client/nodeimage_test.go

```diff
@@ -351,21 +351,21 @@ func TestCapacityPollingHandler(t *testing.T) {
 	assert.Equal(wantCapacity, gotCapacity)
 }
 
-type stubCapacityPoller struct {
```

Comment:
Imo the stub shouldn't be public, is there anything that returns it?

Reply:
I think the issues comes from public functions returning private types?
If so `NewStubCapacityPoller` should be made private, if possible, so we don't export testing stubs

Reply:
That is what I wanted to suggest. :)